Add Git version and OS to issue template (#678)
jupyterlab/jupyterlab-git: .github/issue_template.md

* Look at the README *Troubleshooting* section
* Search for relevant issues

> * Check that you have updated both the jupyterlab extension and the python package to the same version

Reviewer: If adding a hint for how to check git version then there should probably also be a hint for these. Or similar to these command the hint for `git --version` should only appear in the context section

> * Check that your git command has version 2 or higher (using `git --version`)

Reviewer: Better matching the phrasing of the previous bullet point
```suggestion
* Check that you have installed Git version 2 or higher (using `git --version`)
```

<!-- Results of `conda list jupyterlab-git` or `pip show jupyterlab-git` -->
- Extension version:
<!-- Results of `jupyter labextension list` -->
- Git version:
<!-- Results of `git --version` -->

> - OS:

Reviewer: ```suggestion
- Operating System and version:
```
This is more explicit and follows the issue template used by jupyterlab
![image](https://user-images.githubusercontent.com/10111092/84597471-317b8d80-ae32-11ea-8bdc-e7fb43106353.png)
